docs/cli.md

```diff
@@ -8,25 +8,29 @@ last_update:
 license: 'This software is licensed under the Apache License version 2.'
 ---
 
-This document provides guidance on managing metadata within Apache Gravitino using the Command Line Interface (CLI). The CLI offers a terminal based alternative to using code or the REST interface for metadata management.
+This document provides a guidance on managing metadata within Apache Gravitino using
```

Comment:
"a guidance" -> "guidance" as the guidance in the document is not singular

Reply:
OK.